src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1246,6 +1246,20 @@ void VM_Version::get_processor_features() {
   }
 #endif // _LP64
 
+  // Dilithium Intrinsics
+  // Currently we only have them for AVX512
+#ifdef _LP64
+  if (supports_evex() && supports_avx512bw()) {
```

Comment:
supports_evex check looks redundant.

Reply:
These are checks for two different feature bits: CPU_AVX512F and CPU_AVX512BW. Are you saying that the latter implies the former in every implementation of the spec?

Reply:
AVX512BW is built on top of AVX512F spec. In assembler and other places we only check BW in assertions which implies EVEX.